data/campaigns/The_South_Guard/scenarios/08b_The_Tides_of_War.cfg

```diff
@@ -502,12 +504,49 @@
         )}
     [/event]
 
+    [event]
+        name=last breath
+        [filter]
+            id="Mal M'Brin"
+        [/filter]
+
+        [message]
+            speaker="Mal M'Brin"
+            message= _ "Aaargh! You dare strike me?! Puny humans, I am the darkness incarnate! I am power beyond your comprehension!"
```

Comment:
You &lt;i>dare&lt;/i> strike me? Puny humans, I am Darkness incarnate! I wield power beyond your comprehension!